src/templates/trip/leader-trip-card.njk

```diff
@@ -78,7 +78,8 @@ on time. 48 hours before trip-start you can begin checking out your trip.
 <thead>
 <tr>
   <th>Name
-  <th>Attended
+  {% if left === 1 %}<th>Attended{% endif %}
```

Comment:
I see what you're doing here, but it's not necessary. I'd rather the table look the same to every user, every time.